pkg/sql/codegen/xgboost/template.go

```diff
@@ -53,7 +53,8 @@ def xgb_dataset(fn, dataset_sql):
     return xgb.DMatrix(fn)
 
 dtrain = xgb_dataset('train.txt', '''{{.TrainSelect}}''')
-dtest = xgb_dataset('test.txt', '''{{.ValidationSelect}}''')
+# FIXME(weiguoz): bring dtest back when VALIDATE clause is ready
```

Comment:
Sure. Maybe add a check in `Train` to make sure `ValidationSelect` in `TrainIR` is always empty.

Reply:
Will be done in #988